e2e/deploy/workloads/deployments.yaml

```diff
@@ -137,7 +137,7 @@ spec:
       annotations:
         vault.security.banzaicloud.io/vault-addr: "https://vault:8200"
         vault.security.banzaicloud.io/vault-tls-secret: vault-tls
-        vault.security.banzaicloud.io/vault-env-from-path: "secret/data/accounts/aws"
+        vault.security.banzaicloud.io/vault-from-path: "secret/data/accounts/aws"
```

Comment:
just to capture the notes:
we are switching here because we agreed that its okay to switch in reloader project due to low impact footprint. however, even if we dont use the new annotation, this (the old annotation) also works, correct?

Reply:
We haven't agreed on how and whether to preserve backwards compatibility on the webhook.
PR's like this which changes stuff related to the old `vault-secrets-webhook`, should be something to think about after the generic webhook is in place.
I just opened these PR's before, while It was needed to change the Vault CRD in the Operator.

Reply:
this is important because we mentioned preserving the old annotations in the webhook to ensure that things work the same way between versions and we maintain the compatibility. it doesnt matter for us which annotations are being used for these items as long as they are unique (no two same annotations exist, old or new)